src/node_api/model/cli.rs

```diff
@@ -33,9 +33,15 @@ pub struct RequestDockerBuild {
     pub image: String,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct TransparentLogOutputParams {
+    pub format: Option<String>,
```

Comment:
Define `format` as `ContentType` instead of `Option<String>`.
```suggestion
    pub content_type: ContentType,
```